srcpkgs/cudatext-qt5/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'cudatext-qt5'
+pkgname=cudatext-qt5
+version=1.168.6.1
+revision=1
+archs="x86_64"
+hostmakedepends="tar xz python3"
+depends="qt5 python3"
+short_desc="Cross-platform text and code editor, written in Lazarus"
+maintainer="luan-adao <luanadao98@gmail.com>"
+license="MPL-2.0"
+homepage="https://cudatext.github.io"
+distfiles="https://sourceforge.net/projects/cudatext/files/release/${version}/cudatext_${version}-1_qt5_amd64.deb"
+checksum=1a8cb6592a9626a13e4e5d4316aa5f91e020574b7e186814ea16014d722b08b0
+nostrip=yes
+python_version=3.6
```

Comment:
python_version can only be 2 or 3, 3.6 doesn't make sense, we only have python 3.10